var/spack/repos/builtin/packages/aomp/package.py

```diff
@@ -8,6 +8,22 @@
 tools_url = 'https://github.com/ROCm-Developer-Tools'
 compute_url = 'https://github.com/RadeonOpenCompute'
 
+# 3.9 SHA Keys
+aomp39 = dict()
```

Comment:
There's a lot of duplication in this package. Could we use a doubly-nested dict and some for-loops to remove this duplication?